src/app/(BottomTabNavigation)/Profile/DeleteAccount/index.tsx

```diff
@@ -6,16 +6,14 @@ import styles from './styles';
 import WhiteTrash from '../../../../../assets/white-trash.svg';
 import X from '../../../../../assets/x.svg';
 import CasesHeader from '../../../../Components/CasesHeader/CasesHeader';
-import {
-  deleteCurrentUser,
-  getCurrentUserInfo,
-  signOutUser,
-} from '../../../../supabase/queries/auth';
+import { useSession } from '../../../../context/AuthContext';
+import { getCurrentUserInfo } from '../../../../supabase/queries/auth';
```

Comment:
Should this be a function within the `AuthContext`? Ideally we wouldn't have to query this user's information, and it'd already be accessible to them.

Reply:
youre right this should be deleted and not used